src/toniarts/openkeeper/game/state/GameState.java

```diff
@@ -323,6 +322,14 @@ public void onLoadComplete() {
                 GameState.this.stateManager.getState(PlayerState.class).setEnabled(true);
                 GameState.this.stateManager.getState(SoundState.class).setEnabled(true);
 
+                // FIXME PlayerCameraState should initialized before we start
+                while (true) {

```

Comment:
Hmm, got scared there for a moment :) This class is not used.

Reply:
In a far, far-away galaxy, the absence of this code threw an exception. I corrected in the old code so as not to forget why errors occur